packages/nextjs/components/HooksExample.tsx

```diff
@@ -36,6 +38,14 @@ const HooksExample: React.FC = () => {
     contractName: contractName as ContractName,
   });
 
+  const { formatted: ethBalance } = contract
```

Comment:
Hi @bitfalt , regarding the bug, I think you can't really call hooks like this. Hooks cannot be called conditionally since they make it hard to guarantee the rendering order of React. You can make use of the states within the `useScaffoldEthBalance` hook to determine whether the value has been fetched or not (we have loading states too).

<img width="871" alt="image" src="https://github.com/user-attachments/assets/e7cc5d45-5b03-4859-ba6c-411a1ccff8ad">

Rules of Hooks: https://react.dev/reference/rules/rules-of-hooks

Reply:
ohh okay, i didn't see the hint for the hook. I only saw what the hook returned. Thank you so much for addressing my question :D

Reply:
Nice, always good something new sir!